### PR TITLE
Use Jar protoc plugin

### PR DESCRIPTION
**Pull request by jamesward**

### CODE REVIEW

File: run/grpc-hello-world-mvn/pom.xml [hunk 1]
```diff
@@ -80,32 +80,29 @@
       <plugin>
         <groupId>org.xolstice.maven.plugins</groupId>
         <artifactId>protobuf-maven-plugin</artifactId>
-        <version>0.5.0</version>
+        <version>0.6.1</version>
         <configuration>
           <protocArtifact>com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
+          <pluginId>grpc-java</pluginId>
+          <pluginArtifact>io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
```
Comment (bshaffer) [inline run/grpc-hello-world-mvn/pom.xml:87]: isn't this invalid as we are no longer using `exe`?

Comment (jamesward) [reply run/grpc-hello-world-mvn/pom.xml:87]: That is the gRPC Java protoc plugin which doesn't ship a jar: https://repo.maven.apache.org/maven2/io/grpc/protoc-gen-grpc-java/1.32.1/

Comment (bshaffer) [reply run/grpc-hello-world-mvn/pom.xml:87]: ahh, missed that. Thanks!

**Comment by bshaffer** [comment]
I have a fix for the linter in #79. The other failure is happening because this was submitted from a fork.

**Pull request closed without merging**